data/repository/build.gradle.kts

```diff
@@ -48,35 +48,43 @@ kotlin {
                 implementation(projects.domain.authentication)
                 implementation(libs.io.insert.koin.core)
                 implementation(libs.kotlinx.coroutines.core)
+                api(libs.store)
+                implementation ("com.squareup.sqldelight:coroutines-extensions:1.5.4")
             }
         }
 
 
         // android
         getByName("androidMain") {
             dependsOn(commonMain)
-            dependencies {}
+            dependencies {
+                implementation(libs.store)
+            }
         }
 
 
         // desktop
         getByName("desktopMain") {
-            dependencies {}
+            dependencies {
+                implementation(libs.store)
+            }
         }
 
 
         // iOS
         val iosX64Main by getting
         val iosArm64Main by getting
-        val iosSimulatorArm64Main by getting
+//        val iosSimulatorArm64Main by getting
```

Comment:
is this temp? It's best to add some TODO or addition comment with context for any commented out code going into the repo so we don't need to track down PR, comments or discussions to understand why we commented out stuff

Reply:
temp, will fix before landing